ros2param/ros2param/verb/dump.py

```diff
@@ -52,8 +52,8 @@ def add_arguments(self, parser, cli_name):  # noqa: D102
             default='.',
             help='The absolute path were to save the generated file')
         parser.add_argument(
-            '--print', action='store_true',
-            help='Print generated file in terminal rather than saving a file.')
```

Comment:
From a backwards-compatibility standpoint, I think we may want to leave the `--print` option, but just have it print a warning.  That way if someone has scripting where they call this, it will still work in H-Turtle, but it will throw a warning.

Reply:
makes sense.

Reply:
Make sure that the warning would not turn the output into invalid yaml ;)